app/views/curation_concerns/base/_show_actions.html.erb

```diff
@@ -1,7 +1,10 @@
 <div class="form-actions">
   <% if Sufia.config.analytics %>
-    <%= link_to "Analytics", sufia.stats_work_path(@presenter), id: 'stats', class: 'btn btn-default' %>
-  <% end %>
+    <% if @presenter.model_name.name ==  "GenericWork" %>
+      <%= link_to "Analytics", sufia.stats_work_path(@presenter), id: 'stats', class: 'btn btn-default' %>
```

Comment:
@njaffer Ah, right, you won't have access to that "sufia." object outside of views and controllers. I tweaked the code above to reference the paths properly.